### PR TITLE
 (feat) O3-3674 : enhancement on responsiveness for stock management left nav

### PR DESCRIPTION
**Pull request by donaldkibet**
## Requirements
- [x] This PR has a title that briefly describes the work done including a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) type prefix and a Jira ticket number if applicable. See existing PR titles for inspiration.

#### For changes to apps
- [ ]  My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).

#### If applicable
- [ ] My work includes tests or is validated by existing tests.

## Summary
Improve the responsiveness of left nav 

## Screenshots
https://github.com/user-attachments/assets/341ce66a-dd5d-4030-9c13-011b0f4263c3

## Related Issue
https://openmrs.atlassian.net/browse/O3-3674

### CODE REVIEW

File: src/core/components/overlay/overlay.scss [hunk 1]
```diff
@@ -94,4 +95,4 @@
 /* Combo */
 :global(.cds--list-box__menu) {
   z-index: 20000;
-}
+}
```
Comment (hadijahkyampeire) [inline src/core/components/overlay/overlay.scss:98]: Could we add a new line at the end of the file

File: src/dashboard/dashboard-view.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+@use '@carbon/colors';
+
+.dashboardView {
+  background-color: colors.$white-0;
+}
```
Comment (hadijahkyampeire) [inline src/dashboard/dashboard-view.scss:5]: ditto

File: src/dashboard/home-dashboard.scss [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  justify-content: center;
+}
+
+// set up rtl
+[dir='rtl'] {
+  .dashboardContainer {
+    margin-left: 0;
+    margin-right: 16rem;
+  }
+}
```
Comment (hadijahkyampeire) [inline src/dashboard/home-dashboard.scss:38]: ditto

File: src/stock-settings/stock-settings.scss [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    align-items: center;
+
+    &>button {
+        margin-bottom: layout.$layout-03;
+    }
+}
+
+.title {
+    @include type.type-style("heading-01");
+}
```
Comment (hadijahkyampeire) [inline src/stock-settings/stock-settings.scss:22]: ditto

**Review by hadijahkyampeire — COMMENTED** [review]
Thanks @donaldkibet this looks great, just minor comments

**Comment by ojwanganto** [comment]
Thanks, @donaldkibet. Not sure why i am not able to play the attached video.

**Pull request merged**